### ISSUE TITLE
Add directory path clarity to Quickstart Tutorial for new users

### ISSUE THREAD

**Issue by katcaola** [osgi/osgi.enroute.site#195]
I'm a new user to OSGi and I was very confused where to run commands specified in the [quickstart tutorial](https://enroute.osgi.org/tutorial/020-tutorial_qs.html).

In a lot of tech tutorials I've seen, there is a directory path shown before the `$ <CLI COMMAND>` such as `directory/path/location $ ls .`. This helps prevent confusion about where to run CLI commands. 

One issue that took me at least a day to resolve was that the quickstart tutorial told me to[ run commands in the examples/quickstart dir](https://enroute.osgi.org/tutorial/020-tutorial_qs.html#build-and-run) but never told me to run commands anywhere else.

When it came time to [recreate the project using OSGi](https://enroute.osgi.org/tutorial/020-tutorial_qs.html#recreating-quick-start) I didn't know that I had to[ run the project template command outside of the project I had just downloaded](https://enroute.osgi.org/tutorial/020-tutorial_qs.html#project-setup). 
```
user@host:~/sandbox/osgi.enroute/examples/quickstart/quickstart $ mvn org.apache.maven.plugins:maven-archetype-plugin:3.0.1:generate \
-DarchetypeGroupId=org.osgi.enroute.archetype \
-DarchetypeArtifactId=project \
-DarchetypeVersion=7.0.0
user@host:~/sandbox/osgi.enroute/examples/quickstart/ $ ls 
app  pom.xml rest quickstart
```
Since I'm not very `mvn` savvy, I was confused when issues came up about repeated `parent.artifactId` since I was following the instructions exactly as specified. 

```user@host:~/sandbox/osgi.enroute/examples/quickstart/quickstart/impl$ mvn build
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] 'parent.artifactId' must be changed, the parent element cannot have the same
 groupId:artifactId as the project. @ org.osgi.enroute.examples.quickstart:quickstart:1.0-SNAPSHOT,
 /home/user/sandbox/osgi.enroute/examples/quickstart/quickstart/pom.xml, line 5, column 17
```
I would clarify where to run commands for future users by adding detail to the CLI command code snippets:

```
~/osgi.enroute/examples/quickstart $ mvn verify
```
```
~/osgi.enroute/examples/quickstart $ java -jar app/target/app.jar
```
```
~/sandbox $ mvn org.apache.maven.plugins:maven-archetype-plugin:3.0.1:generate \
-DarchetypeGroupId=org.osgi.enroute.archetype \
-DarchetypeArtifactId=project \
-DarchetypeVersion=7.0.0
```

> Having created the project skeleton, edit `~/sandbox/quickstart/impl/src/main/java/org/osgi/enroute/examples/quickstart/rest/ComponentImpl.java`

Etc.

**Comment by timothyjward** [comment]
That looks good to me, as long as we can make sure that the commands are still readable (sometimes the text overflows the side of the page in an awkward way when displaying the commands).

Teel free to create a pull request with the changes.

**Comment by timothyjward** [comment]
That looks fine to me

> I could always put a note about making sure to change the line continuation symbol based on your OS?

A description of the continuations would be a good thing to add in the [prerequisites page](https://github.com/osgi/osgi.enroute.site/blob/62947f0f37628b58e4ffe10340e768e926bb337b/_tutorial/015-Prerequisite.md#conventions). There's also a section on OS specifics which could be used to say what (if anything) different you need to do on Windows/Mac. Putting information there should hopefully avoid us having to call it out every time.

**Comment by katcaola** [comment]
Fixed With Commit #199
